app/controllers/discussions_controller.rb

```diff
@@ -4,7 +4,7 @@ class DiscussionsController < GroupBaseController
   before_filter :we_dont_serve_images_here_google_bot
   before_filter :authenticate_user!, :except => [:show, :index]
   before_filter :load_resource_by_key, except: [:new, :create, :index, :update_version]
-  authorize_resource :except => [:new, :create, :index, :add_comment]
+  authorize_resource :except => [:new, :create, :index, :add_comment, :print]
```

Comment:
Woha!
Big red alert!

You must authorise the resource the same way that the show action does.
Don't merge this until this is fixed. Cheers!